qiskit_nature/mappers/second_quantization/bksf.py

```diff
@@ -0,0 +1,591 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Bravyi-Kitaev Super-Fast (BKSF) Mapper."""
+
+from enum import Enum
+from typing import List, Tuple
+import numpy as np
+
+from qiskit.opflow import PauliSumOp
+from qiskit.quantum_info.operators import Pauli, SparsePauliOp
+from qiskit_nature.operators.second_quantization import FermionicOp
+from .fermionic_mapper import FermionicMapper
+
+
+class BravyiKitaevSuperFastMapper(FermionicMapper):
+    """The Bravyi-Kitaev super-fast fermion-to-qubit mapping.
+
+    This implementation follows closely Reference [1].
+
+       References:
+           [1]: Bravyi-Kitaev Superfast simulation of electronic structure on a quantum computer
+                Kanav Setia and James D Whitfield
+                JCP Volume 148, Issue 16 - Published April 2018
+                arXiv:1712.00446
+    """
+
```

Comment:
I think this needs an init here as below; otherwise it ends up allowing the 2 qubit reduction setting to be changed via base class. Maybe it was a mistake to default it there in the base class.
```
    def __init__(self):
        """The BKSF mapping."""
        super().__init__(allows_two_qubit_reduction=False)
``` 

Maybe this will help with the docs formatting too - as it stands it could do with improving around the way the references come out as it seems a bit too much

![image](https://user-images.githubusercontent.com/40241007/127710846-a8caf813-1865-4a39-83d8-fdff48d2d4b3.png)

Reply:
Yes, the `__init__` method is necessary. Apparently, `allows_two_qubit_reduction` is semantically a class property, but it is implemented on a per-instance basis.